### ISSUE TITLE
Missing portraits

### ISSUE THREAD

**Issue by AnkeSR** [klokantech/bmarchives-tickets#21]
I have recurrently had problems finding portraits on bm archives, here a few examples from my spot check:

http://www.bmarchives.org/items/show/100208982

http://www.bmarchives.org/items/show/100208348

http://www.bmarchives.org/items/show/100208435

Can you help out?

**Comment by klokan** [comment]
You did not describe the problem. What shall we do to reproduce your issue?

**Comment by AnkeSR** [comment]
The digital images of the portraits aren't visible. I just tried Johann Ludwig Krapf and a few more this morning and it seems to be a general problem with the portraits (a recurring problem, if I remember correctly).

**Comment by klokan** [comment]
I expect these photos were never on the bmarchives.org website, because we have not received the original images while importing the initial batch of photos to the web.

Then it is a similar problem as Nature's self printing #13.

If we should fix it then we need you to:
- identify all missing pictures (you speak about "portraits" I expect these were scanned together in a batch and are probably all missing on the web, correct?).
- find the individual digital images (JPEG/TIFF/..?) and ensure their filenames has correct identifiers so we can bind them to metadata records
- deliver the files on a harddisk or online as a zip - for example via Dropbox (BTW I still have your USB disk in the office now)

**Comment by AnkeSR** [comment]
The portraits (one of the keywords linked to the images from the thesaurus) were part of the digitised images on bmpix.org, i.e. part of the whole photographic image collection that was transferred to bmarchives and were not scanned separately as a batch. I have therefore decided to attempt to identify the missing images, I will however need quite some time to identify all the missing images, please be patient. From what I can see so far, most images missing are from the so-called official portrait series with the reference nos. QS-30.001.... I will get back to you on this topic. 
FYI: I am on leave from 26.07.13-05.08.13

**Comment by klokan** [comment]
Note: another alternative way would be to start to collect all the original scans of BMArchives to create the e-Archive - as mentioned in #16:
- First step (BMArchives team): Collecting all the original data (TIFFs) on a physical storage (HDD) and unifying consistently filenames.

**Comment by AnkeSR** [comment]
Yes, that is a very good idea and would probably solve two problems in one. I will speak to Guy to speed up things concerning the e-Archive as soon as I am back from leave.

**Comment by klokan** [comment]
29-11-2013: It is going to be solved together with the digital archive #16.
